public/pages/CreateTrigger/components/TriggerExpressions/TriggerExpressions.js

```diff
@@ -28,34 +69,46 @@ class TriggerExpressions extends Component {
         label={label}
         style={flyoutMode ? { maxWidth: '100%' } : { width: '390px' }}
       >
-        <EuiFlexGroup alignItems={'flexStart'} gutterSize={'m'}>
+        <EuiFlexGroup alignItems="flexStart" gutterSize="m">
           <EuiFlexItem grow={1}>
             <Field name={keyFieldName}>
-              {({ field: { onBlur, ...rest }, form: { touched, errors } }) => (
-                <EuiCompressedFormRow
-                  isInvalid={touched.thresholdEnum && !!errors.thresholdEnum}
-                  error={errors.thresholdEnum}
-                >
-                  <EuiCompressedSelect
-                    options={THRESHOLD_ENUM_OPTIONS}
-                    data-test-subj={`${keyFieldName}_conditionEnumField`}
-                    {...rest}
-                  />
-                </EuiCompressedFormRow>
-              )}
+              {({ field, form, meta }) => {
+                const safeValue = normalizeToSymbol(field.value);
+                if (safeValue !== field.value) {
+                  // Coerce once without triggering validation churn
+                  form.setFieldValue(field.name, safeValue, false);
+                }
+                return (
+                  <EuiCompressedFormRow
+                    isInvalid={!!(meta.touched && meta.error)}
+                    error={meta.error}
+                  >
+                    <EuiCompressedSelect
+                      name={field.name}
+                      value={safeValue}
+                      options={NUMBER_OF_RESULTS_OPERATOR_OPTIONS}
```

Comment:
Just to confirm, `NUMBER_OF_RESULTS_OPERATOR_OPTIONS ` has all of the same entries as `THRESHOLD_ENUM_OPTIONS`, right? I believe bucket monitors use this too, so I want to make sure those aren't impacted.